packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -21,6 +21,8 @@ int _synthesiseDownButtons(int buttons, PointerDeviceKind kind) {
     case PointerDeviceKind.invertedStylus:
       return buttons == 0 ? kPrimaryButton : buttons;
     case PointerDeviceKind.unknown:
+    // ignore: no_default_cases, to allow adding new device types to [PointerDeviceKind].
+    default:
```

Comment:
is this temporary? if so please leave a todo here with a link to the issue where the work is happening

Reply:
(same in the others)

Reply:
Can we make it permanent? Because this enum is always subject to adding new options.

Or what if we just remove `unknown`, since that's what "unknown" is for.

Reply:
removing "unknown" seems fine to me, i don't really understand how "unknown" would ever get set in the first place